.checkstyle/checkstyle.xml

```diff
@@ -98,7 +98,7 @@
         </module>
         <module name="CyclomaticComplexity">
             <!-- default is 10-->
-            <property name="max" value="11"/>
```

Comment:
If you need to override this for some method, you should use annotation and not change the rules for everything.

Reply:
Thanks. I simplify code a little bit and this is not needed anymore :)